src/api/app/controllers/person_controller.rb

```diff
@@ -155,6 +155,15 @@ class NoPermission < APIException
   end
 
   def internal_register
+    if CONFIG['ldap_mode'] == :on
+      render_error(
+        status: 403,
+        errorcode: 'permission_denied',
+        message: "User accounts can not be registered via OBS when in LDAP mode. Please refer to your LDAP server to change it."
```

Comment:
How about "... . Please refer to your LDAP server to create new users." or " to register new users"?